src/main/java/net/syscon/elite/repository/jpa/model/InternetAddress.java

```diff
@@ -0,0 +1,36 @@
+package net.syscon.elite.repository.jpa.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "INTERNET_ADDRESSES")
+public class InternetAddress {
+    @Id
+    @Column(name = "INTERNET_ADDRESS_ID")
```

Comment:
Probably pay to add `nullable = false` to the @Column annotations for those fields that are not nullable (defaults to true).  I appreciate we are not technically generating the schema as the NOMIS DB already exists.

Reply:
sounds like a good thing to document anyway, have added it to the new models